Doc/library/re.rst

```diff
@@ -421,6 +448,20 @@ The special characters are:
    some fixed length.  Patterns which start with negative lookbehind assertions may
    match at the beginning of the string being searched.
 
+``(?>...)``
+  Attempts to match ``...`` as if it was a separate Regular Expression, and if
+  successful, continues to match the rest of the pattern following it.  If the
+  subsequent pattern fails to match, the stack can only be unwound to a point
+  *before* the ``(?>...)`` because once exited, the expression, known as an
+  :dfn:`Atomic Group`, has thrown away all stack points within itself.  Thus,
```

Comment:
Is 'stack point' the same as 'backtrack point'?

Reply:
Not sure yet.